spec/lib/generator_spec.rb

```diff
@@ -37,7 +42,7 @@
       Generator.equipment_model
       Generator.equipment_item
     end
-    it_behaves_like 'generates a valid', :reservation
+    it_behaves_like 'generates multiple valid', :reservation
```

Comment:
we shouldn't do this because sometimes random reservation generation will fail -- we catch the exception because its less complicated than ensuring valid data